modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -325,4 +325,115 @@ spec:
 [source,terminal]
 ----
 $ oc apply -f my-performance-profile.yaml
-----
\ No newline at end of file
+----
+
+[id="node-tuning-operator-pod-power-saving-config_{context}"]
+== Optional: Power saving configurations for pods
+
+You can enable power savings for low priority workloads that are co-located with high priority workloads on an SNO node without impacting the latency or throughput of the high priority workloads. Power saving is possible without modifications to the workloads themselves.
+
+The power saving configurations apply at the pod level, which means that the configuration applies to all the cores used by the pod.
+
+By enabling P-states and C-states, high priority workloads can be configured for best performance and lowest latency in low power mode.
+
+.Power saving configurations
+[cols="1,2", options="header"]
+|====
+|Annotation
+|Description
+
+|`low-latency.crio.io: "enable"`
+|Enables low latency and high performance.
+
+a|[source,terminal]
+----
+annotations:
+`cpu-c-states.crio.io: "enable"`
+`cpu-freq-governor.crio.io: "<governor>"`
+----
+|Allows for flexibility by enabling the lowest C-state in the pod and specifying the governor type for CPU scaling.
```

Comment:
I'd suggest:
Provides the best performance for a pod by disable C-states and specifying the governor type for CPU scaling (the "performance" governor is recommended for high priority workloads).